music21/articulations.py

```diff
@@ -596,27 +595,37 @@ class FretBend(FretIndication):
     Release is the quarterLength value from the start
     of the note for releasing the bend, if Any. Defaults to None.
 
-    WithBar indicates if the bend is done using a whammy bar movement. Defaults to False.
+    WithBar indicates what whammy bar movement is used, if any.
+    MusicXML supports 'scoop' or 'dip'. Defaults to None.
 
-    >>> fb = articulations.FretBend(bendAlter=interval.ChromaticInterval(-2), release=0.5)
+    >>> fb = articulations.FretBend(1,bendAlter=interval.ChromaticInterval(-2),release=0.5)
     >>> fb
-    <music21.articulations.FretBend 0>
+    <music21.articulations.FretBend 1>
     >>> fb.preBend
     False
     >>> fb.withBar
-    False
+    
```

Comment:
In the doctests it's best to show None as one of:

```
>>> fb.withBar is None
True
```

or

```
>>> print(fb.withBar)
None
```

Otherwise the empty space just looks a bit weird.